client/app/nasha/partition/access/add/nasha-partition-access-add.html

```diff
@@ -1,81 +1,49 @@
-<div class="oui-modal oui-modal_shadow nasha-partition-access-add">
-    <form class="form" role="form" data-ng-submit="NashaPartitionAccessAddCtrl.addAccess()" name="nashaPartitionAccessAddForm">
-        <div class="oui-modal__header">
-            <button class="oui-icon oui-icon-close_thin oui-modal__close-button"
-                type="button"
-                aria-label="Exit" data-ng-click="NashaPartitionAccessAddCtrl.dismiss()"></button>
-        </div>
+<form data-ng-submit="NashaPartitionAccessAddCtrl.addAccess()" name="nashaPartitionAccessAddForm"
+    data-ng-validate="NashaPartitionAccessAddCtrl.data.type && NashaPartitionAccessAddCtrl.data.accessToAdd"">
+    <oui-modal
+        data-primary-label="{{ ::'common_add' | translate }}"
+        data-secondary-action="NashaPartitionAccessAddCtrl.dismiss()"
+        data-secondary-label="{{ ::'common_cancel' | translate }}"
+        data-on-dismiss="NashaPartitionAccessAddCtrl.dismiss()"
+        data-loading="NashaPartitionAccessAddCtrl.loading">
 
+        <oui-message type="info">
+            <span data-translate="nasha_partitions_access_notice"></span>
+        </oui-message>
 
-        <div class="oui-modal__body oui-modal__body_no-icon" data-ng-if="NashaPartitionAccessAddCtrl.loading">
-            <div class="text-center">
-                <oui-spinner></oui-spinner>
-            </div>
-        </div>
-        <div class="oui-modal__body oui-modal__body_no-icon" data-ng-if="!NashaPartitionAccessAddCtrl.loading">
-            <h2 class="oui-modal__title" data-translate="nasha_access_action_add"></h2>
+        <h2 class="oui-modal__title" data-translate="nasha_access_action_add"></h2>
```

Comment:
No `h1` above

Reply:
In fact modal has been released to be handled with h2. It should be because h1 is in the header of the page.